### DISCUSSION TITLE
ESM-1b or ESM-2 pre-computed embeddings

### DISCUSSION THREAD

**Discussion by fcampelo** [facebookresearch/esm#478] [Q&A] [answered]
Hi everyone,

Is there a repository somewhere with pre-computed ESM-1b or ESM-2 embeddings? (similar to the ESM Atlas for precomputed structures)?

Cheers,
Felipe

**Comment by tomsercu** [comment, marked as answer]
Not yet, but there's popular demand.. stay tuned

**Reply by lhatsk** [reply]
Thanks for uploading these! I would also be interested in the embeddings for UR50 / UR100, are they available somewhere?

**Reply by chengyou96** [reply]
Are there any updates recently? I'm keen on obtaining the ESM-2 pre-computed embeddings!
